src/clr-angular/forms/datepicker/date-input.ts

```diff
@@ -42,6 +42,22 @@ export class ClrDateInput extends WrappedFormControl<ClrDateContainer> implement
    */
   private _subscriptions: Subscription[] = [];
 
+  //We need this variable because if the date input has a value initialized
+  //we do not output it. This variable is false during initial load. We make sure that
+  //during initial load dayModelOutputted is equal to the value entered by the user so that initialized
+  //value isn't emitted back to the user. After initial load,
+  //we set this to true and the dayModelOutputted is set only
+  //when the Output is emitted to the user.
+  private dayModelOutputtedInitialized: boolean = false;
```

Comment:
is there a reason for not naming it `dayModelOutputInitialized`? Its the `Outputted` part that seems odd to me.

Reply:
I agree with @hippee-lee to keep the names consistent with `dayModelOutput`